assets/app/view/tiles.rb

```diff
@@ -38,7 +38,11 @@ def render_tile_blocks(
         text = tile.preprinted ? '' : '#'
         text += name
         text += "-#{rotation}" unless rotations == [0]
-        text += " × #{num}" if num
+        if tile.unlimited
+          text += ' x ∞'
```

Comment:
It looks like the multiplication sign is different sizes? Should they not be the same?